### ISSUE TITLE
Mix assets component suggestion gets lost when using custom manifest JSON path

### ISSUE THREAD

**Issue by sykorax** [laravel-idea/plugin#36]
Hello,

First of firsts. Thank you for this wonderful plugin!
Using the ```mix()``` helper in the IDE will resolve first parameter components using the lastly built mix manifest json file. This is great so far, however most of my projects includes multiple mix building scripts, so multiple manifest files being created as well.

To use this accordingly, i do the followings (in case i have a portfolio site and an admin / portal site).
```html
<script src="{{ mix('app.js', 'assets/admin') }}"></script>
```
and / or
```html
<script src="{{ mix('app.js', 'assets/site') }}"></script>
```

This way i can have multiple asset json files. Each of them placed in the directory described by the second parameter. (public/assets/admin/mix-manifest.json, public/assets/site/mix-manifest.json)

I was wondering if the plugin could fall back to this kind of resolving strategy in case a default mix-manifest.json could not be found right under the public folder. 
What's your thought? - Sorry for the poor english

**Comment by adelf** [comment]
English is good, don't worry about it.
I'll try to scan all `mix-manifest.json` files and provide great completion for `mix` function parameters :) Thanks.

**Comment by sykorax** [comment]
Hey Adelf,

Great, thank you very much! Awesome :)

**Comment by adelf** [comment]
implemented in 2.4 version. please check and comment if there are some issues.
